neopia/neosoco.py

```diff
@@ -391,6 +391,25 @@ def get_value(self, port: str):
         else:
             raise TypeError
 
+    def led_off(self, port: str):
+        cvt_val = 0

```

Comment:
It's better not using cvt_val variable but, to insert the value, 0 into self.write() is more intuitive to understand codes. Second.